src/tribler/core/components/libtorrent/tests/test_download_manager.py

```diff
@@ -455,3 +492,79 @@ async def test_check_for_dht_ready(fake_dlmgr):
+    fake_metainfo = {b'info': {b'name': b"test_download"}}
+    fake_download.get_def().metainfo = fake_metainfo
+
+    fake_tracker1 = "127.0.0.1/test-announce1"
+
+    fake_dlmgr.update_trackers(fake_download.infohash, [fake_tracker1])
+
+    assert fake_metainfo["announce"] == fake_tracker1
+    assert "announce-list" not in fake_metainfo
+
+
+def test_update_trackers_list(fake_dlmgr) -> None:
+    fake_download, _ = create_fake_download_and_state()
+    fake_dlmgr.downloads[fake_download.infohash] = fake_download
+    fake_metainfo = {b'info': {b'name': b"test_download"}}
+    fake_download.get_def().metainfo = fake_metainfo
+
+    fake_tracker1 = "127.0.0.1/test-announce1"
+    fake_tracker2 = "127.0.0.1/test-announce2"
+
+    fake_dlmgr.update_trackers(fake_download.infohash, [fake_tracker1, fake_tracker2])
+
+    assert "announce" not in fake_metainfo
+    # The order of the results changes between OSes, so this is needed
+    assert (
+        fake_metainfo["announce-list"] == [[fake_tracker1, fake_tracker2]] or
+        fake_metainfo["announce-list"] == [[fake_tracker2, fake_tracker1]]
+    )
+
+
+def test_update_trackers_list_append(fake_dlmgr) -> None:
+    fake_download, _ = create_fake_download_and_state()
+    fake_dlmgr.downloads[fake_download.infohash] = fake_download
+    fake_metainfo = {b'info': {b'name': b"test_download"}}
+    fake_download.get_def().metainfo = fake_metainfo
+
+    fake_tracker1 = "127.0.0.1/test-announce1"
+    fake_tracker2 = "127.0.0.1/test-announce2"
+
+    fake_dlmgr.update_trackers(fake_download.infohash, [fake_tracker1])
+    fake_download.get_def().get_tracker = Mock(return_value=fake_tracker1)
+    fake_dlmgr.update_trackers(fake_download.infohash, [fake_tracker2])
+
+    assert fake_metainfo["announce"] == fake_tracker1
+    # The order of the results changes between OSes, so this is needed
+    assert (
+        fake_metainfo["announce-list"] == [[fake_tracker1, fake_tracker2]] or
+        fake_metainfo["announce-list"] == [[fake_tracker2, fake_tracker1]]
+    )
+
+
+def test_get_download_rate_limit(fake_dlmgr) -> None:
+    fake_value = 42
+    settings = {}
+    fake_dlmgr.get_session = Mock(return_value=Mock(get_settings=Mock(return_value=settings)))
+    fake_dlmgr.get_session().download_rate_limit = functools.partial(settings.get, "download_rate_limit")
+    fake_dlmgr.ltsessions = {0: fake_dlmgr.get_session()}
+    fake_dlmgr.set_download_rate_limit(fake_value, 0)
+
+    assert fake_value == fake_dlmgr.get_download_rate_limit()
+
+
+def test_get_upload_rate_limit(fake_dlmgr) -> None:
+    fake_value = 42
+    settings = {}
+    fake_dlmgr.get_session = Mock(return_value=Mock(get_settings=Mock(return_value=settings)))
+    fake_dlmgr.get_session().upload_rate_limit = functools.partial(settings.get, "upload_rate_limit")
+    fake_dlmgr.ltsessions = {0: fake_dlmgr.get_session()}
+    fake_dlmgr.set_upload_rate_limit(fake_value, 0)
+
+    assert fake_value == fake_dlmgr.get_upload_rate_limit()
```

Comment:
For asserts it is common practice to use the following syntax `assert actual==expected`
```suggestion
    assert fake_dlmgr.get_upload_rate_limit()==fake_value
```